consensus/tests/common.rs

```diff
@@ -242,17 +242,29 @@ mod tests {
         let mut chain_a = ProtocolStateChain::default();
         let mut consensus_state = ConsensusState::default();
         consensus_state.blockchain_length = Length(117896);
-        consensus_state.last_vrf_output =
-            VrfOutputTruncated::from("ZYmm1mr8vJ6F-1pOKgmF1yAF41Z9onq1Je5PoKSBAwA=");
+        // BUGBUG: This should be base64 decoded
```

Comment:
Yes, fixing it as part of recent changes in https://github.com/ChainSafe/mina-rs/pull/172

Reply:
blake2b decoded

Reply:
It's a blake2b hash btw: https://github.com/MinaProtocol/mina/blob/compatible/docs/specs/consensus/README.md#516-hashlastvrf